HelpSource/Classes/LFSaw.schelp

```diff
@@ -20,7 +20,8 @@ argument::iphase
 
 Initial phase offset. For efficiency reasons this is a value
 ranging from 0 to 2.
-
+Note:: enter "1" for an initial phase of 0 radiants.
+A value of 0 would start the cycle at pi radiants. See the example below.::
```

Comment:
this is correct, but sounds illogical. Maybe:

Note:: By default, LFSaw starts at code::-1:: and goes to code::1:: Use a phase offset of code::1:: to start at code::0:: (midways).
?